Framework/PythonInterface/mantid/utils/absorptioncorrutils.py

```diff
@@ -48,13 +48,20 @@ def __get_instrument_name(wksp):
     return mantid.kernel.ConfigService.getInstrument(ws.getInstrument().getName()).shortName()
 
 
-def __get_cache_name(meta_wksp_name, abs_method, cache_dirs=[], prefix_name=""):
+def __get_cache_name(
+    meta_wksp_name,
+    abs_method,
+    cache_dirs=[],
+    prefix_name="",
+    ms_method="",
+):
     """generate candidate cachefile names (full paths) and associated ascii hash
 
     :param meta_wksp_name: name of workspace contains relevant meta data for hashing
     :param abs_method: method used to perform the absorption calculation
```

Comment:
add a note about this being required if the ms_method is supplied. Maybe it is better to supply that information on the ms_method parameter docs

Reply:
the docstring is updated as suggested